src/lib.rs

```diff
@@ -125,6 +125,7 @@ pub enum Terminator {
     Call {
         operand: CallOperand,
         cleanup_bb: Option<BasicBlockIndex>,
+        ret_bb: Option<BasicBlockIndex>,
```

Comment:
How does this change work in isolation? Wouldn't it have either a) not compiled before this b) not compile now?

Reply:
I'll update the commit message to explain.